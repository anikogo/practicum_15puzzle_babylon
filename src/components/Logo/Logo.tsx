import classnames from 'classnames';

export default function Logo() {
  return (
    <span
      className="inline-flex justify-start items-center gap-2 text-orange-500 dark:text-green-500 group-hover:text-green-300"
    >
      <span className="rounded overflow-hidden">
        <span
          className={classnames(
            'border-2 border-gradient-br-indigo',
            'bg-orange-500 dark:bg-green-500 group-hover:bg-green-300',
            'grid aspect-square w-12 rounded place-items-center',
            'text-white text-3xl font-bold',
          )}
        >
          15
        </span>
      </span>
      <span className="uppercase font-bold text-xl">Puzzle</span>
    </span>
  );
}
