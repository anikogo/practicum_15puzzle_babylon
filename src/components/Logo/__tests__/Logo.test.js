/* eslint-disable jsx-a11y/anchor-is-valid */
import renderer from 'react-test-renderer';
import Logo from '../Logo';

it('logo', () => {
  const component = renderer.create(
    <Logo />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
