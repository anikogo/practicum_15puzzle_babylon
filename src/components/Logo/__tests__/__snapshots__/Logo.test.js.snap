// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<span
  className="inline-flex justify-start items-center gap-2 text-green-500 group-hover:text-green-300"
>
  <span
    className="rounded overflow-hidden"
  >
    <span
      className="border-2 border-gradient-br-indigo bg-green-500 group-hover:bg-green-300 grid aspect-square w-12 rounded place-items-center text-white text-3xl font-bold"
    >
      15
    </span>
  </span>
  <span
    className="uppercase font-bold text-xl"
  >
    Puzzle
  </span>
</span>
`;

exports[`logo 1`] = `
<span
  className="inline-flex justify-start items-center gap-2 text-green-500 group-hover:text-green-300"
>
  <span
    className="rounded overflow-hidden"
  >
    <span
      className="border-2 border-gradient-br-indigo bg-green-500 group-hover:bg-green-300 grid aspect-square w-12 rounded place-items-center text-white text-3xl font-bold"
    >
      15
    </span>
  </span>
  <span
    className="uppercase font-bold text-xl"
  >
    Puzzle
  </span>
</span>
`;
